src/styles/main.scss

```diff
@@ -12,16 +23,20 @@ html {
 body {
     display: flex;
     flex-direction: column;
-    font-size: 1.4rem;
+    font-size: 1.8rem;
+    -moz-osx-font-smoothing: grayscale;
+    -webkit-font-smoothing: antialiased;
```

Comment:
Why `antialiased` instead of the default or `subpixel-antialiased`?  Subpixel antialiasing looks much better on retina displays, while antialiasing looks better on non-retina.  By default, the OS/browser will choose what it thinks is best.  Usually this only needs to be set on specific elements that are using a scale transform to avoid a pseudo-text-flickering.